samples/benchmark/gamelife/kotlin/gradle.properties

```diff
@@ -0,0 +1 @@
+#kotlin.native.jvmArgs=-agentlib:jdwp=transport=dt_socket,server=y,suspend=y,address=5006
```

Comment:
Not sure this has to be added

Reply:
not necessarily
this is for debugging the entry code generation